docs/docs/concepts.mdx

```diff
@@ -522,120 +522,6 @@ for modifying **multiple** key-value pairs at once:
 
 For key-value store implementations, see [this section](/docs/integrations/stores/).
 
-### Tools
```

Comment:
@rlancemartin shall we keep the section headings everywhere to maintain a glossary on the main concepts page?

We can remove the beefy content, but maintain headings so no links break